frontend/src/component/changeRequest/ChangeRequest/SegmentTooltipLink.tsx

```diff
@@ -70,9 +71,10 @@ export const SegmentTooltipLink: FC<IStrategyTooltipLinkProps> = ({
                     maxHeight: 600,
                 }}
             >
-                <Typography component="span">
-                    {formatStrategyName(change.payload.name)}
```

Comment:
Not sure why we were calling `formatStrategyName` here, but I suspect it was a copy-paste error that was left in.